eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOinput.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2022 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+/**
+ * EO org.eolang.io package.
+ *
+ * @since 0.27.0
+ * @checkstyle PackageNameCheck (4 lines)
+ */
+package EOorg.EOeolang.EOio;
+
+import java.util.Scanner;
+
+/**
+ * All system inputs.
+ *
+ * @since 0.28.0
+ */
+public final class EOinput {
+    /**
+     * Default input.
+     */
+    private static EOinput instance;
+
+    /**
+     * Scanner.
+     */
+    private Scanner scanner;
+
+    /**
+     * Ctor.
+     */
+    private EOinput() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * GetInstance.
+     * @return The pointer to input
+     */
+    public static synchronized EOinput getInstance() {
```

Comment:
@includealex Have you considered creating an `Input` object in the global scope using an enum?

Reply:
@Graur No, I have not.

Reply:
@yegor256 Can we use here `enum` as global scope object, or just leave it as is (aka `singletone`) ?